pkg/controllers/manila/manila.go

```diff
@@ -123,6 +145,65 @@ func (c *ManilaController) sync(ctx context.Context, syncCtx factory.SyncContext
 	return c.setEnabledCondition(ctx)
 }
 
+func (c *ManilaController) getFsGroupPolicy(ctx context.Context) storagev1.FSGroupPolicy {
+
+	// Set the fsGroupPolicy param to the default value
+	fsGroupPolicy := "None"
+
+	// Try to get the current value defined for the operator
+	if c.isFSGroupPolicyAllowed(os.Getenv("CSI_FSGROUP_POLICY")) {
+		fsGroupPolicy = os.Getenv("CSI_FSGROUP_POLICY")
+	}
+
+	return storagev1.FSGroupPolicy(string(fsGroupPolicy))
+}
+
+func (c *ManilaController) applyCSIDriver(ctx context.Context, spec storagev1.CSIDriverSpec) error {
+	klog.V(4).Infof("Starting CSI driver config refresh")
+	defer klog.V(4).Infof("CSI driver config refresh finished")
+
+	var errs []error
+	crd := c.generateCSIDriver(spec)
+	_, _, err := resourceapply.ApplyCSIDriver(ctx, c.kubeClient.StorageV1(), c.eventRecorder, crd)
+	if err != nil {
+		errs = append(errs, err)
+	}
+	return k8serrors.NewAggregate(errs)
+}
+
+func (c *ManilaController) generateCSIDriver(csiOpt storagev1.CSIDriverSpec) *storagev1.CSIDriver {
+
+	crd := &storagev1.CSIDriver{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "manila.csi.openstack.org",
+			Annotations: map[string]string{
+				"csi.openshift.io/managed": "true",
+			},
+		},
+		Spec: csiOpt,
+	}
```

Comment:
Please read static yaml manifest from `assets/` and change only `fsGroupPolicy` here - I'd like to have all objects in one place (`assets/`) and not to look around various places.

Reply:
That object is removed from assets/ because the spec field is computed in a function according to the env defined in the operator manifest. 
Restoring assets/ makes this PR pointless.
I can close it if this approach doesn't match the way the operator is managed.

Reply:
Some parts of assets/controller.yaml (and other yaml files) are computed too. Still, the template is in assets and only the computed parts are modified in go code. I'd really like to have all the templates in assets and only set the env. var in go code.

Reply:
Ok, thank you for the comment and the suggestion. 
After digging a bit more, I realized that you can still have the `csidriver.yaml` asset statically defined, and put some logic in library-go as done for `StorageClasses` [1]  to make sure the the whole object is recreated in case an "immutable" field changes.
`FsGroupPolicy` is one of them: we can build a generic static CSIDriver object, with the default value, and recreate it in case it changes.
I'll push an update to reflect this behavior, which is the same pattern I see for the `StorageClass`.

[1] https://github.com/openshift/library-go/commit/354d53d860bff27de6adbf72dfaa4812a9fff61d